### ISSUE TITLE
development branch prediction conflicts with sklearn.metrics 

### ISSUE THREAD

**Issue by Motorrat** [automl/auto-sklearn#156]
I have done pip install git+https://github.com/automl/auto-sklearn@development
and re-run my binary classification code that completes with no error. Now the sklearn.metrics.accuracy_score crashes when fed prediction results from autosklearn:
```
c.show_models()
y_pred = c.predict(X_test.values)
```
y_train and y_test are coming from train_test_split(X,y) so they should be of the same type (numpy.ndarray). And I see that type(y_pred) is numpy.ndarray too and its of the the length of 6600 (=len(y_test)).

so here is the error message:
```
    print("Accuracy score", accuracy_score(y_test, y_pred))
  File "/x/Redly/anaconda3/lib/python3.5/site-packages/sklearn/metrics/classification.py", line 172, in accuracy_score
    y_type, y_true, y_pred = _check_targets(y_true, y_pred)
  File "/x/Redly/anaconda3/lib/python3.5/site-packages/sklearn/metrics/classification.py", line 74, in _check_targets
    type_pred = type_of_target(y_pred)
  File "/x/Redly/anaconda3/lib/python3.5/site-packages/sklearn/utils/multiclass.py", line 260, in type_of_target
    not isinstance(y.flat[0], string_types)):
IndexError: index 0 is out of bounds for axis 0 with size 0
```

**Comment by Motorrat** [comment]
Interesting enough, after rolling back the autosklearn to 0.0.2 the F1 for the exactly same dataset and code has increased from 0.34 to 0.38. Whatever has happened with 0.0.2 since my last update from the autosklearn main branch - pls more of that :-)

**Comment by mfeurer** [comment]
Thanks for reporting this. We've found the error and are working on a fix. What improved auto-sklearn when downgrading is something that I can't answer...

**Comment by mfeurer** [comment]
This is going to be fixed in #167

**Comment by mfeurer** [comment]
Fixed. There will be a release this afternoon, so it'll be available from pypi.
